criu/config.c

```diff
@@ -528,6 +528,7 @@ int parse_options(int argc, char **argv, bool *usage_error,
 		{ "status-fd",			required_argument,	0, 1088 },
 		BOOL_OPT(SK_CLOSE_PARAM, &opts.tcp_close),
 		{ "verbosity",			optional_argument,	0, 'v'	},
+		{ "binary-file",		optional_argument,	0, 'b' },
```

Comment:
It should be "no_argument", since it's a boolean option. With the respective short option change.